rcl_action/src/rcl_action/action_client.c

```diff
@@ -752,6 +752,41 @@ rcl_action_client_set_status_subscription_callback(
     user_data);
 }
 
+#define CLIENT_CONFIGURE_SERVICE_INTROSPECTION(TYPE, STATE) \
+  if (rcl_client_configure_service_introspection( \
+      &action_client->impl->TYPE ## _client, \
+      node, \
+      clock, \
+      type_support->TYPE ## _service_type_support, \
+      publisher_options, \
+      STATE) != RCL_RET_OK) \
+  { \
+    ret = RCL_RET_ERROR; \
```

Comment:
or maybe, so that the caller does not possess `ret` variable in the scope.

```suggestion
#define CLIENT_CONFIGURE_SERVICE_INTROSPECTION(RET, TYPE, STATE) \
  if (rcl_client_configure_service_introspection( \
      &action_client->impl->TYPE ## _client, \
      node, \
      clock, \
      type_support->TYPE ## _service_type_support, \
      publisher_options, \
      STATE) != RCL_RET_OK) \
  { \
    RET = RCL_RET_ERROR; \
```

Reply:
Based on the changes above, it returns directly, so there's no need for `ret`.